### PR TITLE
Added sound master control registers to mmio_addresses.

### LINKED ISSUE #143: `SOUNDCNT_L`/`SOUNDCNT_H` definitions are missing

### PR DESCRIPTION
**Pull request by Lymia**
Fixes #143.

### CODE REVIEW

File: src/mmio_types/fifo_enable.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+use super::*;
+
+#[derive(Debug, Clone, Copy, Default, PartialEq, Eq)]
+#[repr(transparent)]
+pub struct FifoEnable(u32);
```
Comment (Lokathor) [inline src/mmio_types/fifo_enable.rs:5]: I think this should be just a u8, which saves on size if this is placed into a struct.

I also think we should have entries for the Sound ON checking bits. Even though they're read only, a person still might want to read them of course.

File: src/mmio_types/sound_bias.rs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+use super::*;
+
+#[derive(Debug, Clone, Copy, Default, PartialEq, Eq)]
+#[repr(transparent)]
+pub struct SoundBias(u32);
+impl SoundBias {
```
Comment (Lokathor) [inline src/mmio_types/sound_bias.rs:6]: This can be a u16 type

File: src/mmio_types/sound_status.rs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+use super::*;
+
+#[derive(Debug, Clone, Copy, Default, PartialEq, Eq)]
+#[repr(transparent)]
+pub struct SoundStatus(u32);
+impl SoundStatus {
```
Comment (Lokathor) [inline src/mmio_types/sound_status.rs:6]: a u8 type

**Review by Lokathor — CHANGES_REQUESTED** [review]

File: src/mmio_types/fifo_enable.rs (hunk 1 above)
Comment (Lymia) [reply src/mmio_types/fifo_enable.rs:5]: Looking at how you handled other stuff, I split the same register into `FifoEnable` (which is R/W) and `SoundStatus` (which is read-only). Do you think it'd be better to just have a single R/W register definition that handles both?

**Review by Lokathor — APPROVED** [review]

File: src/mmio_types/fifo_enable.rs (hunk 1 above)
Comment (Lokathor) [reply src/mmio_types/fifo_enable.rs]: I'll accept the PR as-is for now, and we can have a look at this later. 

Opened: https://github.com/rust-console/gba/issues/148

**Pull request merged**